pytorch_to_returnn/torch/nn/modules/operator.py

```diff
@@ -273,12 +273,14 @@ class Gather(Module):
   """
   is_original_torch_module = False
 
-  def __init__(self, dim: int, pos: int):
+  def __init__(self, dim: int, pos: Union[int, Tensor]):
```

Comment:
I'm not sure but maybe it's a bad idea (inconsistent) to have tensors as arguments in the `__init__`?

Reply:
Maybe it makes more sense to have a separate module (`Slice` or so) for this case, where `pos` is then an argument for `create_returnn_layer_dict` and not for `__init__`.

Reply:
I pushed this now.